x-pack/plugins/dashboard_enhanced/public/services/drilldowns/dashboard_drilldowns_services.ts

```diff
@@ -44,14 +44,23 @@ export class DashboardDrilldownsService {
     { advancedUiActions: uiActions }: SetupDependencies
   ) {
     const start = createStartServicesGetter(core.getStartServices);
+    const getDashboardUrlGenerator = () => {
+      const urlGenerator = start().plugins.dashboard.dashboardUrlGenerator;
+      if (!urlGenerator)
+        throw new Error('dashboardUrlGenerator is required for dashboard to dashboard drilldown');
```

Comment:
This message contains error message specific for drilldowns.

> ... required for dashboard to dashboard drilldown

When somebody else in this plugin will start using the dashboard URL generator, this error message will not make sense for them.

Reply:
But it is inside `/dashboard_drilldowns_services.ts` 🤔